pkg/endpoint/endpoint.go

```diff
@@ -2007,3 +2007,106 @@ func (e *Endpoint) Delete(monitor monitorOwner, ipam ipReleaser, manager endpoin
 
 	return errs
 }
+
+type ContainerStartFunc func()
```

Comment:
Did you have more to this signature in the future PRs? Since it's used in one place I would prefer to just have
`func (e *Endpoint) Create(ctx context.Context, containerStartFunc func(), syncBuild bool)`
(or `containerStartCB` or whatever). The `ContainerStartFunc` will be public but will show up somewhere else in the docs so it isn't as clear what the signature should be when trying to call `Endpoint.Create` (this may be an IDE fail thing, shrug).

Reply:
Yeah, makes sense. I'll just pass a `func()` directly.